derive-new-test/Cargo.lock

```diff
@@ -1,38 +0,0 @@
-[root]
```

Comment:
Thanks for clarifying! It does make sense indeed (as long as *everyone is doing back-compat properly*...).